Kconfig.nrf

```diff
@@ -11,9 +11,23 @@ menu "Nordic nRF Connect"
 config BOARD_THINGY91_NRF9160NS
 	bool
 
-# nRF Connect SDK needs a larger default stacks for running certain tests.
+# Handle default to ensure that BT only will disable ENTROPY_CC3XX,
+# while multi protocol does not.
+config ENTROPY_CC3XX
+	default y if ZIGBEE || OPENTHREAD_NRF_SECURITY
+	default n if BT
+
+config ZIGBEE
+	select ENTROPY_CC3XX
```

Comment:
Why redefine the config ZIGBEE here? Shouldn't this select statement be put in nrf/subsys/zigbee?

Reply:
This has been removed. It was added in error